docs/modules/write-the-docs/pages/write-docs.adoc

```diff
@@ -336,10 +342,10 @@ Where possible, include the source file in the `./images` directory as well; thi
 
 Insert an image into a document using the following syntax:
 
-[source]
+[source, asciidoc]
 ----
 .This is the image caption
-image::docs/02_opennms-logo.png[]
+image::docs/02_opennms-logo.png[opennms logo]
 ----
 
 The image path for all images that you include is relative to the `.adoc` file where the image is referenced.
```

Comment:
Hmmm... Maybe we should have a discussion about this. On one hand, I can see how it can be considered redundant, since we have a table of modules in the Style Guide already, but on the other hand it is nice to be able to see where all of the image folders are and how they line up. 

@Bonrob2, what are your thoughts?

Reply:
Looking at the diagram this morning, I realized we'd have to keep it updated, and Mark's point is valid. Let's remove it.